components/dialog/stories/dialog.stories.js

```diff
@@ -3,47 +3,71 @@ import { disableDefaultModes } from "@spectrum-css/preview/modes";
 import { isOpen } from "@spectrum-css/preview/types";
 import { Template as Typography } from "@spectrum-css/typography/stories/template.js";
 import { version } from "../package.json";
+// import { DialogGroup } from "./dialog.test.js";
```

Comment:
You can remove this comment if you don't need this anymore.

Reply:
Yes, thank you! There are a few things in Dialog I kept immediately after the rebase just in case I needed them, but it looks like I don't! Removed: https://github.com/adobe/spectrum-css/pull/2833/commits/e9310f3a16d6a47215ef76f57b11e545c3da9c3b